Apps/WebClient/src/NewClientApp/src/stores/auth.ts

```diff
@@ -70,9 +69,6 @@ export const useAuthStore = defineStore("auth", () => {
         idpHint?: string
     ): Promise<void> {
         try {
-            // EventBus.$emit(
```

Comment:
How are you managing the waitlist's ticket release process when navigating away from the site for the Keycloak login.

Reply:
`authService.signIn` redirects only when the user doesn't have a valid token.  When returning to the site after signing in, or refreshing the page while signed in, `signIn` is called but will not trigger a redirect.

Unregistering the listeners now happens in `restAuthService.ts` when navigation to the OIDC sign in or sign out page is guaranteed.